dmutils/status.py

```diff
@@ -47,3 +48,69 @@ def enabled_since(date_string):
         return date_string
 
     return False
+
+
+class StatusError(Exception):
+    """A stub class to use when implementing additional checks for an app's _status endpoint. See API for example."""
+    message = ''
+
+    def __init__(self, message):
+        super().__init__()
```

Comment:
You should probably pass on `message` to the base constructor, otherwise `str()` conversion doesn't work as expected for an `Exception`.

Reply:
Done 👍